### PR TITLE
Bubble up conflicts

### LINKED ISSUE #3222: Inform User about Conflict File creation
Hi,
Currently, when the conflict file is created, it is not synchronized with the server, and the user is not warned.
To prevent the user having to check every time you save a file (to check if it was synchronized), I suggest that a warning message is displayed to the user that the file was not synchronized and why.

Thanks!

### PR DESCRIPTION
**Pull request by dragotin**
This pull request shows a tray message if conflicts happened. fixes #3222

### CODE REVIEW

File: src/gui/folder.cpp [hunk 1]
```diff
@@ -511,6 +520,13 @@ void Folder::createGuiLog( const QString& filename, LogStatus status, int count,
                 text = tr("%1 has been moved to %2.").arg(file).arg(renameTarget);
             }
             break;
+        case LogStatusConflict:
+            if( count > 1 ) {
+                text = tr("%1 has and %2 other files have sync conflict.").arg(file, count-1);
```
Comment (ogoffart) [inline src/gui/folder.cpp]: I don't thik this prints what you think it prints.  the second parameter to your arg is the fieldWidth.  Also the text reads weird.
I'd write:

tr("%1 and %n other files have sync conflict.", "", count-1).arg(file)

File: src/gui/folder.cpp [hunk 2]
```diff
@@ -511,6 +520,13 @@ void Folder::createGuiLog( const QString& filename, LogStatus status, int count,
...
+            } else {
+                text = tr("%1 has a sync conflict. Please check the conflict file!").arg(file);
+            }
+            break;
         case LogStatusError:
             if( count > 1 ) {
                 text = tr("%1 and %2 other files could not be synced due to errors. See the log for details.", "%1 names a file.").arg(file).arg(count-1);
```
Comment (ogoffart) [inline src/gui/folder.cpp]: I realize this has the same problem!

**Comment by dragotin** [comment]
@ogoffart thanks, I pushed a fix.

**Comment by ogoffart** [comment]
:+1:  code looks good to me.

**Comment by guruz** [comment]
:+1:

**Pull request merged**